components/table/filterDropdown.tsx

```diff
@@ -266,6 +266,8 @@ class FilterMenu<T> extends React.Component<FilterMenuProps<T>, FilterMenuState<
         confirm: this.handleConfirm,
         clearFilters: this.handleClearFilters,
         filters: column.filters,
+        visible: this.getDropdownVisible(),
+        getPopupContainer: (triggerNode: HTMLElement) => triggerNode.parentNode as HTMLElement,
```

Comment:
This line seems to cause an error in some highly customized dropdown filters as `triggerNode.parentNode` is being `null` for some reason on initial component mount.